src/data_generators/heterogeneous_impression_generator.py

```diff
@@ -29,7 +29,7 @@ def __init__(
         n: int,
         gamma_shape: float = 1,
         gamma_scale: float = 1,
-        random_state: RandomState = None,
+        random_generator: np.random.Generator = None,
     ):
         """Constructor for the HeterogeneousImpressionGenerator.
 
```

Comment:
Not related to the changes. I think the term in exp() in the pdf of the Gamma distribution in the docstring should have a negative sign.

Reply:
Fixed in PR#37. Thanks Rieman.